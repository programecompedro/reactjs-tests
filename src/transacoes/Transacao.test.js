import Transacao from "./Transacao"
import React from 'react';
import { render } from "@testing-library/react"

describe("Componente Transacao", ()=>{

    describe("Snapshot do componente", () => {
        test("deve permanecer o mesmo", () => {
            const { container } = render(<Transacao data="15/10/2021" tipo="saque" valor="20"/>)

            expect(container.firstChild).toMatchSnapshot();
        })
    })
})