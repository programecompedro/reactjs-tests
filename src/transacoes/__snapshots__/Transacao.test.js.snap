// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Componente Transacao Snapshot do componente deve permanecer o mesmo 1`] = `
<div
  class="transacao-container"
>
  <p>
    15/10/2021
  </p>
  <p>
    saque
  </p>
  <p>
    R$ 
    20
  </p>
</div>
`;
